### PR TITLE
WEBEU1 - Sprint Challenge User Interface Elisa Martin 

### PR DESCRIPTION
**Pull request by elisamartin**

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -12,57 +12,95 @@
...
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+      <img class="logo" src="img/lambda-black.png" alt="Lambda logo">
+        <nav>
+          <a href="index.html">Home</a>
+          <a href="about.html">About</a>
+          <a href="#">Products</a>
+          <a href="#">Blog</a>
+          <a href="#">Contact</a>
+        </nav>
+
```
Comment (samarv) [inline about.html:28]: You can use something like [Prettier]( https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) to easily style your code.

File: about.html [hunk 2]
```diff
@@ -12,57 +12,95 @@
...
+        <a href="#" class= "more">Learn More</a>
+      
+      </div>
+      
+      <div class="about-section"> 
+        <img class="img-section" src="img/about-working.png" alt="strategy">
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+        <h2>How We Work</h2>
+
```
Comment (samarv) [inline about.html:58]: You MUST NOT leave a white space after every line.

File: about.html [hunk 3]
```diff
@@ -12,57 +12,95 @@
...
+  <div class="container about-page">
 
+    <header>
 
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+      <img class="logo" src="img/lambda-black.png" alt="Lambda logo">
```
Comment (samarv) [inline about.html:20]: Style it so, it aligns with nav bar.

File: css/index.css [hunk 4]
```diff
@@ -143,4 +209,49 @@ footer nav {
...
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: baseline;
+}
+
+.more{
+    border: 2px solid black;
+    border-radius: 10px;
+	text-align: center;
```
Comment (samarv) [inline css/index.css:243]: -.-  I am going to be a stickler here but please use prettier and be conscious of your code style.

**Review by samarv — COMMENTED** [review]
## Overview


**MVP**

* [X] Build the HTML and CSS to create the missing navigation and header.
* [X] Link the `About` navigation item to the [about.html](about.html) page
* [X] Color Boxes
* [X] Copy and paste your home page navigation and header into the about page
* [X] Update the header image with the about page image
* [X] Link the `Home` navigation item back to the `index.html` page.
* [X] Build the rest of the about page layout to match the design

**Stretch**

* [ ] Build a page of your choosing from the navigation items.  Come up with content and images that fit the theme.  
* [ ] Introduce CSS animations to your site.
* [ ] Build a contact page and create a form with several inputs of your choosing
* [ ] Add responsive breakpoints to your code by using media queries


**Conclusion** 


Elisa, Your overall code is clean and to the point.  You have created a pixel perfect website according to the design specs. There are few things, I want you to work on 

1. Code Style - check out [HTML5 Style Guide](https://www.w3schools.com/html/html5_syntax.asp)

2. Git commits - You want your messages to be more granular and more regular. [git style guide](https://udacity.github.io/git-styleguide/)



In conclusion, You did a superb job! By creating this pixel perfect website. You have shown mastery of the Week 1 of Lambda School. Keep doing the good work. 


 I am really looking forward to seeing more work from you. and I think that if you keep up with this quality of work across the board you will do fine here at lambda 👍 *remember that if any doubt about process or procedure we are always available in the help channel and feel free to reach out to me for any further explanations or clarifications of anything said in this review*
